packages/react-components/react-textarea/src/components/Textarea/useTextareaStyles.styles.ts

```diff
@@ -32,6 +32,7 @@ const useRootStyles = makeStyles({
     backgroundColor: tokens.colorTransparentBackground,
     ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStrokeDisabled),
     [`& > textarea`]: {
```

Comment:
Ideally we  should not be using a `& > textarea` selector here. Instead, these should be on a `disabled` class in `useTextareaStyles` and applied directly to the slot.

I know this isn't part of this change, so it doesn't need to block this PR, and could be fixed in a separate PR instead.

Reply:
Created an issue to track that work #27838